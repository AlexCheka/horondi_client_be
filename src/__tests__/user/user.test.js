/* eslint-disable no-undef */
const { gql } = require('apollo-boost');
const userService = require('../../modules/user/user.service')
const userResolver = require('../../modules/user/user.resolver')
const client = require('../../utils/apolloClient');

require('dotenv').config();


describe('querries', () => {
    test('should recive all users like in service and resolver', async () => {
        const res = await client.query({
            query: gql`
                query {       
                    getAllUsers{
                        email
                        firstName
                        lastName
                        phoneNumber
                        purchasedProducts
                        role
                        orders
                        wishlist
                        credentials{
                            source
                            tokenPass
                        }
                        registrationDate
                        address{
                            country
                            city
                            street
                            appartment
                            buildingNumber
                        }
                    }
                }
            `
        }).then(res => {
            return res;
        })
        .catch(error => {
          console.log(error)
        });

        expect(userService.getAllUsers()).resolves.toBe(res);
        expect(userResolver.userQuery.getAllUsers()).resolves.toBe(res);
        expect(res.data.getAllUsers).toMatchSnapshot()
    })

//     test('should recive user by id like in service and resolver', async () => {
//       const res = await client.query({
//           query: gql`
//               query {       
//                 getUserByToken{ 
//                     name{
//                         lang
//                         value
//                     }
//                     description{
//                         lang
//                         value
//                     }
//                     images{
//                         large
//                         medium
//                         small
//                         thumbnail
//                     }
//                     material
//                     handmade
//                     available
//                 }
//               }
//           `,
//           context: {
//             headers: {
//                 token: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZWZjYWY3MWU2MWYwYjQ3MDA4NzA2MTIiLCJlbWFpbCI6InRhY2prYTM0QGdtYWlsLmNvbSIsImlhdCI6MTU5NDIyMjY3MywiZXhwIjoxNTk0MjI2MjczfQ.ECGiqT1ubWP6UWe3hbpH57-USJUvlw25-ZFNhz4NOZ8"
//             }
//           }
//     }).then(res => res)
//     .catch(err => err)

//     console.log(res)
      
//     //   expect(userService.getUserByToken("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZWZjYWY3MWU2MWYwYjQ3MDA4NzA2MTIiLCJlbWFpbCI6InRhY2prYTM0QGdtYWlsLmNvbSIsImlhdCI6MTU5NDIxOTMxNSwiZXhwIjoxNTk0MjIyOTE1fQ.f2RbXNM0HN7sgjrDwcUgN5edT2m6QV5590EnyKPKbic")).resolves.toBe(res);
//     //   expect(userResolver.userQuery.getUserById('','',{headers:{
//     //       token: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZWZjYWY3MWU2MWYwYjQ3MDA4NzA2MTIiLCJlbWFpbCI6InRhY2prYTM0QGdtYWlsLmNvbSIsImlhdCI6MTU5NDIxOTMxNSwiZXhwIjoxNTk0MjIyOTE1fQ.f2RbXNM0HN7sgjrDwcUgN5edT2m6QV5590EnyKPKbic"
//     //   }})).resolves.toBe(res);
//     //   expect(res.data.getUserById).toMatchSnapshot()
//   })
})