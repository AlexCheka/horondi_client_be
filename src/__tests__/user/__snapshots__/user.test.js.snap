// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mutations should authorize and recive user token 1`] = `
Object {
  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI1ZWZjYWY3MWU2MWYwYjQ3MDA4NzA2MTIiLCJlbWFpbCI6InRhY2prYTM0QGdtYWlsLmNvbSIsImlhdCI6MTU5NDIyMjI5NiwiZXhwIjoxNTk0MjI1ODk2fQ.wu4q6mWQb3N3K7uJDdYCUhXIzlZtq6uaWATnm1TT2ME",
}
`;

exports[`querries should recive all users like in service and resolver 1`] = `
Array [
  Object {
    "address": Object {
      "appartment": "50",
      "buildingNumber": "0",
      "city": "Глухів",
      "country": "Україна",
      "street": "Москворецька вулиця",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "QAGZ6XWOIM",
      },
      Object {
        "source": "facebook",
        "tokenPass": "IB4NDCH32A",
      },
    ],
    "email": "cscjsh5bmr@gmail.com",
    "firstName": "Тамірлан",
    "lastName": "Філліпс",
    "orders": Array [],
    "phoneNumber": "380631922040",
    "purchasedProducts": Array [],
    "registrationDate": "1503648956295",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "2",
      "buildingNumber": "245",
      "city": "Бахчисарай",
      "country": "Україна",
      "street": "Вулиця Ползунова",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "P6W5ODUBZA",
      },
    ],
    "email": "3ulm1kelkk@gmail.com",
    "firstName": "Йоана",
    "lastName": "Решетняк",
    "orders": Array [],
    "phoneNumber": "380394705308",
    "purchasedProducts": Array [],
    "registrationDate": "1479152952630",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "128",
      "buildingNumber": "54",
      "city": "Бурштин",
      "country": "Україна",
      "street": "Вірменський провулок",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "QBCG5B7NX1",
      },
    ],
    "email": "d4ywuv5w0d@gmail.com",
    "firstName": "Сільвія",
    "lastName": "Барбоза",
    "orders": Array [],
    "phoneNumber": "380634053124",
    "purchasedProducts": Array [],
    "registrationDate": "1324093967519",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "36",
      "buildingNumber": "76",
      "city": "Здолбунів",
      "country": "Україна",
      "street": "Вулиця Гребінки",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "LLJ5FXG6JB",
      },
    ],
    "email": "j4676rmr7u@gmail.com",
    "firstName": "Віктоія",
    "lastName": "Холодна",
    "orders": Array [],
    "phoneNumber": "380635336524",
    "purchasedProducts": Array [],
    "registrationDate": "1427873923532",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": Object {
      "appartment": "15",
      "buildingNumber": "118",
      "city": "Кіцмань",
      "country": "Україна",
      "street": "Вулиця Громова",
    },
    "credentials": Array [
      Object {
        "source": "facebook",
        "tokenPass": "85L8KOVPUT",
      },
    ],
    "email": "rylimy3u27@gmail.com",
    "firstName": "Евелін",
    "lastName": "Клебанов",
    "orders": Array [],
    "phoneNumber": "380931978217",
    "purchasedProducts": Array [],
    "registrationDate": "1436372213987",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "76",
      "buildingNumber": "22",
      "city": "Алушта",
      "country": "Україна",
      "street": "Вулиця Ковпака",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "815N6XTFY6",
      },
      Object {
        "source": "facebook",
        "tokenPass": "B03PMZ8V7K",
      },
    ],
    "email": "iag4stiruk@gmail.com",
    "firstName": "Умарджон",
    "lastName": "Маліцький",
    "orders": Array [],
    "phoneNumber": "380969673154",
    "purchasedProducts": Array [],
    "registrationDate": "1462739297737",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "54",
      "buildingNumber": "123",
      "city": "Немирів",
      "country": "Україна",
      "street": "Вулиця Шолом-Алейхема",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "9M8GT1G9EG",
      },
      Object {
        "source": "facebook",
        "tokenPass": "TUGM7AROTT",
      },
    ],
    "email": "70sibdk2x7@gmail.com",
    "firstName": "Макар",
    "lastName": "Ідельсон",
    "orders": Array [],
    "phoneNumber": "380984803586",
    "purchasedProducts": Array [],
    "registrationDate": "1290114082221",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "84",
      "buildingNumber": "128",
      "city": "Скалат",
      "country": "Україна",
      "street": "Рожнівська вулиця",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "I5OJ3N2YW9",
      },
      Object {
        "source": "facebook",
        "tokenPass": "S9EHWD05OE",
      },
    ],
    "email": "idfq78uvpr@gmail.com",
    "firstName": "Єремій",
    "lastName": "Фісун",
    "orders": Array [],
    "phoneNumber": "380502578487",
    "purchasedProducts": Array [],
    "registrationDate": "1434893443975",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "42",
      "buildingNumber": "175",
      "city": "Алушта",
      "country": "Україна",
      "street": "Соборна площа",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "YEJO9WNAQI",
      },
      Object {
        "source": "facebook",
        "tokenPass": "2L6ITI7JKP",
      },
    ],
    "email": "bpk4yi6tnv@gmail.com",
    "firstName": "Лівія",
    "lastName": "Візиренко",
    "orders": Array [],
    "phoneNumber": "380997682291",
    "purchasedProducts": Array [],
    "registrationDate": "1275045309256",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": Object {
      "appartment": "51",
      "buildingNumber": "128",
      "city": "Кобеляки",
      "country": "Україна",
      "street": "Прирічна вулиця",
    },
    "credentials": Array [
      Object {
        "source": "facebook",
        "tokenPass": "EI2A3VZA9T",
      },
    ],
    "email": "efinsg83w7@gmail.com",
    "firstName": "Дорофій",
    "lastName": "Вериківський",
    "orders": Array [],
    "phoneNumber": "380676437154",
    "purchasedProducts": Array [],
    "registrationDate": "1403877783681",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "136",
      "buildingNumber": "120",
      "city": "Черкаси",
      "country": "Україна",
      "street": "Вулиця Саврасова",
    },
    "credentials": Array [
      Object {
        "source": "facebook",
        "tokenPass": "IZ0THJZU54",
      },
    ],
    "email": "fj1hhdjzps@gmail.com",
    "firstName": "Ігор",
    "lastName": "Манн",
    "orders": Array [],
    "phoneNumber": "380503896669",
    "purchasedProducts": Array [],
    "registrationDate": "1443736637533",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "54",
      "buildingNumber": "74",
      "city": "Старий",
      "country": "Україна",
      "street": "Студентська вулиця",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "F6V01F026X",
      },
      Object {
        "source": "facebook",
        "tokenPass": "BYMMW4R4AI",
      },
    ],
    "email": "1x0si2qbrx@gmail.com",
    "firstName": "Лукіян",
    "lastName": "Зельдич",
    "orders": Array [],
    "phoneNumber": "380957992404",
    "purchasedProducts": Array [],
    "registrationDate": "1556857894460",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "96",
      "buildingNumber": "4",
      "city": "Копичинці",
      "country": "Україна",
      "street": "Вулиця Миколи Гоголя",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "6FSNR5ES9Q",
      },
    ],
    "email": "fbulxrje4e@gmail.com",
    "firstName": "Крисандер",
    "lastName": "Вірченко",
    "orders": Array [],
    "phoneNumber": "380639613360",
    "purchasedProducts": Array [],
    "registrationDate": "1442929035885",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "98",
      "buildingNumber": "283",
      "city": "Вугледар",
      "country": "Україна",
      "street": "Вулиця Урожайна",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "XQF3KDBGHR",
      },
    ],
    "email": "re9r6x1qsi@gmail.com",
    "firstName": "Ардіан",
    "lastName": "Веллер",
    "orders": Array [],
    "phoneNumber": "3806881960",
    "purchasedProducts": Array [],
    "registrationDate": "1575591665724",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": Object {
      "appartment": "74",
      "buildingNumber": "253",
      "city": "Гірське",
      "country": "Україна",
      "street": "Вулиця Гетьмана Мазепи",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "2F2FLSHEXL",
      },
    ],
    "email": "lc0yaw7qkn@gmail.com",
    "firstName": "Павел",
    "lastName": "Задорожний",
    "orders": Array [],
    "phoneNumber": "380961884736",
    "purchasedProducts": Array [],
    "registrationDate": "1314385519733",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": Object {
      "appartment": "45",
      "buildingNumber": "104",
      "city": "Борислав",
      "country": "Україна",
      "street": "Лісорубна вулиця",
    },
    "credentials": Array [
      Object {
        "source": "google",
        "tokenPass": "22DC033OYF",
      },
    ],
    "email": "meirj0n6de@gmail.com",
    "firstName": "Емілія",
    "lastName": "Тідеман",
    "orders": Array [],
    "phoneNumber": "380393148853",
    "purchasedProducts": Array [],
    "registrationDate": "1554678336667",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": Object {
      "appartment": "27",
      "buildingNumber": "247",
      "city": "Волочиськ",
      "country": "Україна",
      "street": "Майдан Свободи",
    },
    "credentials": Array [
      Object {
        "source": "facebook",
        "tokenPass": "JU3MRGYSG7",
      },
    ],
    "email": "u33vx0ds3r@gmail.com",
    "firstName": "Руслана",
    "lastName": "Брантман",
    "orders": Array [],
    "phoneNumber": "380985282442",
    "purchasedProducts": Array [],
    "registrationDate": "1551561309950",
    "role": "user",
    "wishlist": Array [
      "f94fe36ef59ff80fe02afb28",
    ],
  },
  Object {
    "address": Object {
      "appartment": "108",
      "buildingNumber": "130",
      "city": "Святогірськ",
      "country": "Україна",
      "street": "Вулиця Шевченка",
    },
    "credentials": Array [
      Object {
        "source": "local",
        "tokenPass": "IGB7TNMCOJ",
      },
    ],
    "email": "4fse4k049j@gmail.com",
    "firstName": "Томас",
    "lastName": "Абдураїмов",
    "orders": Array [],
    "phoneNumber": "380671905326",
    "purchasedProducts": Array [],
    "registrationDate": "1268485215271",
    "role": "user",
    "wishlist": Array [
      "c2a6b03f190dfb2b4aa91f8a",
    ],
  },
  Object {
    "address": Object {
      "appartment": "145",
      "buildingNumber": "290",
      "city": "Бурштин",
      "country": "Україна",
      "street": "Площа Некрасова",
    },
    "credentials": Array [
      Object {
        "source": "facebook",
        "tokenPass": "SL2WMEAVM7",
      },
    ],
    "email": "kh5am6axjg@gmail.com",
    "firstName": "Деміан",
    "lastName": "Кручинін",
    "orders": Array [],
    "phoneNumber": "380503547495",
    "purchasedProducts": Array [],
    "registrationDate": "1332059204637",
    "role": "user",
    "wishlist": Array [
      "da31e5c576af0a3f93888aac",
    ],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$UoVJ6p1IzPNaq7jQnPKe4OIQ3oC9fG2vhrO5M5j2Y/xVfLh46CZEC",
      },
    ],
    "email": "bla@bla.bla",
    "firstName": "blabla",
    "lastName": "blabla",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593251604490",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$YHK2T5TpaaAywP5e7A.6Z.qRN1aj0qZhT2gZ1NrsJGDcCHamVLIeK",
      },
    ],
    "email": "qwerty@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593327748095",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$MzgmdttbgaTL.TX7PNLUa.5EHWMSgvvuICXyngGnoNWYtdWWqkzs.",
      },
    ],
    "email": "bandvov@gmail.com",
    "firstName": "Aa",
    "lastName": "Aa",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593363906080",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$V4vzwt8e7KbRREDTjGnMZuXLFwbBn7uUg8.nD5xS9Q6Gd2nQqFjp6",
      },
    ],
    "email": "qwerty123@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593374597665",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$2kmc3rzHCongq0j.PzGs0OveB7Uz3cvIGxHK/fQiIHnY/KMdkIvHa",
      },
    ],
    "email": "helloworld@gmail.com",
    "firstName": "Sara",
    "lastName": "Connor",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593437506308",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$YoS4AR0M2CwtAqyRx5VFw.lg/JbX/PNgOCVptB2uVDa0fWL/OTYKO",
      },
    ],
    "email": "vovka.trach@gmail.com",
    "firstName": "Hello",
    "lastName": "World",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593438960431",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$NvDXIUejmF3TZWIpoS0JBusqQhx0xemonPQZCtjpavRKjYXEwglqK",
      },
    ],
    "email": "vovrach@gmail.com",
    "firstName": "COVID",
    "lastName": "Vrach",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593458625238",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$7l/7P8wTn9H5DAOfXkXACeU2OuIjzdNGZzZwxYqHmQYuGlmqamVQ.",
      },
    ],
    "email": "4@ghdgsg.com",
    "firstName": "gdhsjsjs",
    "lastName": "hgdfshg",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593460164608",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$b4JwjAG37IiMZCbvdPQbQeiwsalwwudy49orDBEODcjkjxwsbRshS",
      },
    ],
    "email": "Test1111@gmail.com",
    "firstName": "Testtest",
    "lastName": "Testtest",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593508168709",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$P2sBl.zLJ/OSfU9FQJBbCOQBqazq8f3gd/oPvtSdKwxPKM2Rkv.l6",
      },
    ],
    "email": "hld@gmail.com",
    "firstName": "Indiana",
    "lastName": "Johns",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593508683505",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$ia4JxR.dbZQs6AmpQugFb.GbWK6czuxhJCqcyc9hhV9BMEVDdaaMG",
      },
    ],
    "email": "Testuser@gmail.com",
    "firstName": "Testuser",
    "lastName": "Testuser",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593509359212",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$1IfzuSj47IkU/t2pwZLmq.KxJfHzJYfjpz9f9IZLK0lNbPoq2pdAq",
      },
    ],
    "email": "qwertyTest123@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593517204101",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$S8VXPbfL1uerZhhzp84nVec537Y1hgJ2u1oz6Zg6F3o1iGRjnzt0e",
      },
    ],
    "email": "tacjka34@gmail.com",
    "firstName": "Petro",
    "lastName": "Tatsenyak",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593618289150",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$zXyQGpMtbdeEGz6d1MRUNuzYnXexusQICoqYTrXNy5RgmhjoWJfiW",
      },
    ],
    "email": "qwertyqwerty@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1593687049233",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$806vaqY0hpgDb7SrNxeWnO3srj5ymDJs7G06/EGgQiaZ02.p7qWJq",
      },
    ],
    "email": "refactor@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1594203945159",
    "role": null,
    "wishlist": Array [],
  },
  Object {
    "address": null,
    "credentials": Array [
      Object {
        "source": "horondi",
        "tokenPass": "$2a$12$TlisC9kW8EVsRmLSTMgCGu2CJzAkBoNcbRLOdC4NaWxUu6B.SXDOq",
      },
    ],
    "email": "aaaaa@gmail.com",
    "firstName": "qwerty",
    "lastName": "qwerty",
    "orders": Array [],
    "phoneNumber": null,
    "purchasedProducts": Array [],
    "registrationDate": "1594206569398",
    "role": null,
    "wishlist": Array [],
  },
]
`;
